samples/config.mk

```diff
@@ -10,7 +10,7 @@ else ifneq ($(shell $(CC) --version | grep clang),)
         # CC is default (cc), and aliases to clang.
 else
         # CC is default (cc), and does not alias to clang.
-        CLANG_VERSION = $(shell for v in "9" "8" "7"; do \
+        CLANG_VERSION = $(shell for v in "9" "8"; do \
```

Comment:
This file only goes up to v9, but scripts/lvi-mitigation/install_lvi_mitigation_bindir  goes up to v11, and clang itself is up to v12 (released  November 2020).   How do we get consistency in terms of what the latest version of clang is referenced in OE files?